service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageService.kt

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.messaging.message
+
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.Forbidden
+import fi.espoo.evaka.shared.domain.NotFound
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class MessageService {
+    fun replyToThread(
```

Comment:
This function could probably be just a top level function in `Message.kt`. At least right now I can't think of any benefit of making it a service unless maybe if you want to mock it or use dependency injection.